map_view/display-layer-view-state/src/main/java/com/esri/samples/display_layer_view_state/DisplayLayerViewStateSample.java

```diff
@@ -70,8 +71,12 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create an ArcGISMap with the topographic basemap
-      ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create an ArcGISMap with the topographic basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
 
       // create a map view and set the ArcGISMap to it
```

Comment:
Sneaky `ArcGISMap ` here as well ( I won't comment if there's any other instances):

```suggestion
      // create a map view and set the map to it
```